src/review/entities/review.entity.ts

```diff
@@ -0,0 +1,49 @@
+import { IsInt, IsString, IsOptional, Min, Max } from 'class-validator';
+import {
+  Column,
+  Entity,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  JoinColumn,
+  CreateDateColumn,
+  Unique,
+} from 'typeorm';
+import { User } from '@app/common';
+import { Guide } from 'src/guide/entities/guide.entity';
+
+@Entity('Review')
+@Unique(['userId', 'guideId'])
+export class Review {
+  @PrimaryGeneratedColumn()
+  reviewId: number;
+
+  @Column()
+  @IsInt()
+  userId: number;
+
+  @Column()
+  @IsInt()
+  guideId: number;
+
+  @Column({ type: 'int' })
+  @IsInt()
+  @Min(1, { message: 'Rating must be at least 1 star' })
+  @Max(5, { message: 'Rating must be at most 5 stars' })
+  stars: number;
```

Comment:
Just call it rating instead of stars.

Reply:
I think that stars is a better naming, it is more explanatory